### ISSUE TITLE
Add the paper of the specified author from the previous day

### ISSUE THREAD

**Issue by zzezze** [TideDra/zotero-arxiv-daily#22]
Can you add the most recent papers published by the specified author [if there is a duplicate name, use the adding institution]? This will also solve the problem with Google Scholar subscriptions, because it is relatively troublesome to subscribe to and view each author on Google Scholar using an email address.

**Comment by TideDra** [comment]
you can add the `au` field in the arxiv query to filter by authors, while it can not handle duplicate names. It is difficult to distinguish duplicated names because we cannot get much information about the authors from the paper metadata, even the pdf file, which is hard to parse in a unified manner. I personally think the best way to follow an specified author is to follow his social media :D
